src/components/CreateGameForm.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+const Card = styled.section`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 400px;
+  height: 300px;
+
+  p {
```

Comment:
if you agreed to use styled components, then you should avoid using nested styles, and create separate component for each element